scripts/test-operator-integration.sh

```diff
@@ -23,7 +23,7 @@ MOM_CMD="${MOM_CMD:-multi-operator-manager}"
 
 # Define input and output directories (can be overridden if necessary)
 APPLY_CONFIG_INPUT_DIR="${APPLY_CONFIG_INPUT_DIR:-./test-data/apply-configuration}"
-APPLY_CONFIG_OUTPUT_DIR="${APPLY_CONFIG_OUTPUT_DIR:-./test-output}"
+APPLY_CONFIG_OUTPUT_DIR="${ARTIFACT_DIR:-./test-output}"
```

Comment:
This sets `APPLY_CONFIG_OUTPUT_DIR` to the value of `ARTIFACT_DIR` if it is set (which is the case in CI). If `ARTIFACT_DIR` is not set, it uses `./test-output` as the default.

/lgtm